spec/unit/middleware/rate_limiter_v2_api_spec.rb

```diff
@@ -68,6 +68,25 @@ module Middleware
             _, response_headers, _ = middleware.call(user_2_env)
             expect(response_headers['X-RateLimit-Remaining-V2-API']).to eq('80')
           end
+
+          context 'the user is exempt from v2 rate limiting' do
```

Comment:
I would move this context out of `describe 'X-RateLimit-Remaining-V2-API' do` and `describe 'headers as regular user' do` as it seems misplaced to me. Instead it would make a good top level context.